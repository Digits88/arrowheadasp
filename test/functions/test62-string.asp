<!-- Test String function.
-->

String(1, 65) = <%= String(1, 65) %><br>
String(2, 65) = <%= String(2, 65) %><br>
String(10, 65) = <%= String(10, 65) %><br>
String(0, 65) = <%= String(0, 65) %><br>
String(12, "a") = <%= String(12, "a") %><br>
String(12, "z") = <%= String(12, "z") %><br>
String(12, "chacha") = <%= String(12, "chacha") %><br>
