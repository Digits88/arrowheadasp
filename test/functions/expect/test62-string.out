<!-- Test String function.
-->

String(1, 65) = A<br>
String(2, 65) = AA<br>
String(10, 65) = AAAAAAAAAA<br>
String(0, 65) = <br>
String(12, "a") = aaaaaaaaaaaa<br>
String(12, "z") = zzzzzzzzzzzz<br>
String(12, "chacha") = cccccccccccc<br>
